<launch>    

    <!-- Configuration of Jackal which you would like to simulate. -->
    <arg name="ns" default="" /> 

    <!-- ANCHORS SERVER -->
    <node pkg="jackal_range" name="UGV_vis_line" type="UGV_vis_line.py" >
        <param name="odom" type="string" value="/$(arg ns)ground_truth/state" />        
        <param name="params_name" type="string" value="/AnchorsInfo" />
        <param name="topic" type="string" value="/$(arg ns)vis_line" />
    </node>  

</launch>